src/pypg/traits/observable.py

```diff
@@ -86,10 +88,8 @@ def __init__(
         self._canceled = False
         self._data_lock = Lock()
         self._on_error = on_error
-        self.data_event = Event()
```

Comment:
data event is no longer used. it was only relevant to unblock the persistent thread.